oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticReliabilityTest.java

```diff
@@ -86,9 +74,9 @@ protected ElasticConnection getElasticConnection() {
     }
 
     @Test
-    @Ignore("OAK-10558")
```

Comment:
Is the flakiness of the test resolved? What was the problem?

Reply:
yes, the explanation is in the PR description